src/go/networking/cmd/proxy/wsl_integration_linux.go

```diff
@@ -91,5 +91,7 @@ func setupLogging(logFile string) {
 
 	if debug {
 		logrus.SetLevel(logrus.DebugLevel)
+	} else {
+		logrus.SetLevel(logrus.InfoLevel)
```

Comment:
With the fix to `-debug` is this still needed?

It doesn't hurt, though, so I have no issues with leaving it in.

(Note that we probably want to squash commits if we revert this.)